### DISCUSSION TITLE
podman - Error: error getting default registries to try: short-name "vaultwarden/server:latest"

### DISCUSSION THREAD

**Discussion by rapkin61** [dani-garcia/vaultwarden#2123] [Q&A]
### Subject of the issue

Tried to run vaultwarden using podman as described in the wiki.
But podman does not know how to pull the image. As far as I understood, it needs a FQDN 

podman run -d --name vaultwarden -v /var/vw_data/:/data/:Z -e ROCKET_PORT=8080 -p 8080:8080 vaultwarden/server:latest
Error: error getting default registries to try: short-name "vaultwarden/server:latest" did not resolve to an alias and no unqualified-search registries are defined in "/etc/containers/registries.conf"

What is the correct path to the image?

Moreover, podman seems to need some kind of lingering, like:

sudo loginctl enable-linger 1012

where 1012 is the UID of the user that runs the image

### Deployment environment

* vaultwarden version:

* Install method:

* Clients used: 

* Reverse proxy and version: 

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

**Comment by davidkna** [comment]
The FQDN is `docker.io/vaultwarden/server:latest`. `docker.io` is usually listed in `/etc/containers/registries.conf` by default.

**Comment by rapkin61** [comment]
I'm using Debian bullseye (on a raspberry), but there is no <code>docker.io</code> in <code>/etc/containers/registries.conf</code>. This file is completely commented out - i.e. it is empty in fact. 
Using <code>docker.io/vaultwarden/server:latest</code> works perfect - thank you!
